tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/TensorMapper.java

```diff
@@ -0,0 +1,35 @@
+package org.tensorflow;
+
+import org.tensorflow.internal.c_api.TF_Tensor;
+import org.tensorflow.types.family.TType;
+
+/**
+ * Maps the native memory of a {@link RawTensor} to a n-dimensional typed data space
+ * accessible from the JVM.
+ *
+ * <p>Usage of this class is reserved for internal purposes only.
```

Comment:
We might want to have this in a different package as the module system operates on a package level (so when we go to 11 we could not export that package). Though it depends how the methods are used.

Reply:
One of the goal of this abstract class is to be able to access the native handle of the raw tensor from a `TensorMapper` subclass, this is done via `nativeTensor(RawTensor)` and it needs to reside in the `org.tensorflow` package. 

Unless we want to make `RawTensor.nativeHandle()` public (which I would prefer to avoid) or go back to the previous design where both `RawTensor` and `TF_Tensor` are passed to the mapper but that triggered a series of other concerns as discussed in my [previous PR](https://github.com/tensorflow/java/pull/160/files#r535330352).

Reply:
Ok. I definitely don't want the native handle method to be public. I feel like there is probably a way around this, but it might be more trouble than it's worth, so leaving it here is fine.

Reply:
Split packages are definitely something you want to avoid at all costs. These are going to be OSGi users complaining about this as well.

Reply:
Well we could reflectively access a private method as we'd be in the same module. But that's asking for trouble and sets a bad precedent. There's probably something we could do that's a bit tricksy with inner classes, but I can't currently see all the way through it, and again it's probably more trouble than it's worth.